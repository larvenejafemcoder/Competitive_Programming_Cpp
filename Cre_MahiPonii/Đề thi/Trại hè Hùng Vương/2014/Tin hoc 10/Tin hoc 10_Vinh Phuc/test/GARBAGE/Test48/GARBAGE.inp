ENNNWNENNDWDNUNEDUSUWNDDDEWUWNSESDWWWWWEDDUEWNNNEUNWUUNWNUDUENUEEWSNNUNSSENUDWUESSDUNDN
NUUEUENDWSNEUWDSDNWEEWWWDNWSEUWWEUNWWSUNEDDUUDNWSWNUEWUSUS
DUSSEWDDENDDSENDNDNWWEWSSEUUD
NSWWSSWESSWSNSWNSEWUWNUEEWDWWEDSUWSENSNEDDUDSWSNSSNNNDUEWWNENENEWDDUUWDSNEDWEDDNNNW
WEDDNDWNSEWNEW
WDWWNEUESWDWEEUWEEEDNDEWD
E 74
