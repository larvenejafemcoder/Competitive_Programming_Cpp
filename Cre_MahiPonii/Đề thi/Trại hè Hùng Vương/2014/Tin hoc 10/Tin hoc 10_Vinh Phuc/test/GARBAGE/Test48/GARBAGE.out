73259418354259904247067118463168339832858321932280004454664577622067363584019351081297202717729482095167028790123821063338691
